vscode-dotnet-runtime-library/src/Acquisition/DotnetPathFinder.ts

```diff
@@ -129,12 +129,36 @@ Bin Bash Path: ${os.platform() !== 'win32' ? (await this.executor?.execute(Comma
 
         const findCommand = CommandExecutor.makeCommand(pathLocatorCommand, ['dotnet']);
         const dotnetsOnPATH = (await this.executor?.execute(findCommand, options))?.stdout.split('\n').map(x => x.trim()).filter(x => x !== '');
-        if(dotnetsOnPATH)
+        if(dotnetsOnPATH && dotnetsOnPATH.length > 0)

```

Comment:
nit: Just a random Q. Why is `PATH` in all caps here and in other constants below?

Reply:
It's the PATH environment variable which is generally in caps :) Its easier to tell I mean the env var if I use PATH instead of Path which could be other paths. At least imo. Thanks for your review.